drivers/spi/spi_litespi.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2019 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SPI_LITESPI__H
+#define _SPI_LITESPI__H
+
+#include "spi_context.h"
+
+#include <sys/sys_io.h>
+#include <device.h>
+#include <drivers/spi.h>
+
+#define SPI_BASE_ADDR DT_INST_0_LITEX_SPI0_BASE_ADDRESS
+#define SPI_CONTROL_ADDR SPI_BASE_ADDR
+#define SPI_STATUS_ADDR (SPI_BASE_ADDR + 0x08)
+#define SPI_MOSI_DATA_ADDR (SPI_BASE_ADDR + 0x0c)
+#define SPI_MISO_DATA_ADDR (SPI_BASE_ADDR + 0x10)
+#define SPI_CS_ADDR (SPI_BASE_ADDR + 0x14)
+#define SPI_LOOPBACK_ADDR (SPI_BASE_ADDR + 0x18)
```

Comment:
so these above are registers? Maybe better mentioning it in the name (with _REG added?)

Reply:
Replaced `_ADDR` suffix with `_REG` suffix, e.g. `SPI_BASE_REG` for clarity.